schedule.html

```diff
@@ -0,0 +1,322 @@
+      <h2 class="footnote">
+        Sponsors
+      </h2>
+      <article class="sponsors">
+        <h3 id="platinum-sidebar">
+          Platinum Sponsor
+        </h3>
+        <p>
+          <a href="https://www.google.com/"><img src="google.svg" alt=
+          "Google"></a><br>
+          <a href="https://www.mozilla.org/"><img src="mozilla.svg" alt=
+          "Mozilla"></a>
+        </p>
+      </article>
+      <p>
+        <a href="./#sponsorship">Become a sponsor!</a>
+      </p>
+      <p>
+        <a href="sponsorship.html">Sponsorship package info.</a>
+      </p>
+    </aside>
+    <main>
+      <h3 id="schedule">
+        Tentative Schedule
+      </h3>
+      <p>
+        The workshop will focus around several topics identified by the
+        expressions of interest. Each topic will be introduced by one or more
+        related lightning talks, and will be explored more in-depth by
+        discussion breakouts, concluded with joint summaries of the breakouts.
+        The goal of the discussion is not be to resolve the technical issues of
+        the topic, but to determine its relevance and priority to
+        standardization.
+      </p>
+      <p>
+        <b>This schedule may change based on discussions with the program
+        committee.</b>
+      </p>
+      <h4 id="day1">
+        Day 1: 19 October
+      </h4>
+      <table class="schedule">
+        <tbody>
+          <tr>
+            <th>
+              08:00–08:30
+            </th>
+            <td>
+              Registration
+            </td>
+          </tr>
+          <tr>
+            <th>
+              08:30–08:45
+            </th>
+            <td>
+              Opening remarks by Dominique Hazael-Massieux
+            </td>
+          </tr>
+          <tr>
+            <th>
+              08:45–09:00
+            </th>
+            <td>
+              Keynote: (to be announced)
+            </td>
+          </tr>
+          <tr>
+            <th>
+              09:00–09:45
+            </th>
+            <td>
+              WebVR intro: implementation status, obstacles, future plans
+            </td>
+          </tr>
+          <tr>
+            <th class="break">
+              09:45–10:00
+            </th>
+            <td>
+              Break
+            </td>
+          </tr>
+          <tr>
+            <th>
+              10:00–11:00
+            </th>
+            <td>
+              VR user interactions in browsers
+            </td>
+          </tr>
+          <tr>
+            <th>
+              11:00–12:00
+            </th>
+            <td>
+              Accessibility of VR experiences
+            </td>
+          </tr>
+          <tr>
+            <th class="break">
+              12:00–13:00
+            </th>
+            <td>
+              Lunch
+            </td>
+          </tr>
+          <tr>
+            <th>
+              13:00–14:00
+            </th>
+            <td>
+              Shared VR experiences
```

Comment:
this is a discussion topic (what's needed to build remote multi-user VR experiences), not an activity